rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormUserDataProvider.groovy

```diff
@@ -47,6 +48,7 @@ class GormUserDataProvider implements UserDataProvider {
 
     @Override
     @Transactional
+    @Synchronized
```

Comment:
can we remove this? I think it doesn't solve the issue, but may be misleading to someone who thinks it does solve it.